test/core/derivatives/derivatives_test.py

```diff
@@ -101,8 +101,8 @@ def test_jac_t_mat_prod(problem, V=3):
 )
 @pytest.mark.parametrize(
     "problem",
-    PROBLEMS_WITH_WEIGHTS,
-    ids=IDS_WITH_WEIGHTS,
+    PROBLEMS_WITH_WEIGHTS + CONVOLUTION_TRANSPOSED_FAIL_PROBLEMS,
+    ids=IDS_WITH_WEIGHTS + CONVOLUTION_TRANSPOSED_FAIL_IDS,
```

Comment:
Rename `FAIL_` to `GROUPS_` as they don't fail anymore, but contain grouped transpose convolutions